### ISSUE TITLE
Request: Allow extendedInMain to work for helmets and the bauble, too.

### ISSUE THREAD

**Issue by MalkContent** [McJty/TheOneProbe#255]
I'd really like it if I didn't have to press sneaking to see the extended information when I'm wearing the The One Probe helmets/bauble.
The reason being that I have bound a numpad key to show the probe information in the first place and it's kind of awkward to press L-shift at the same time, because i have to take the hand away from the mouse for that. :c

**Comment by kevindevm** [comment]
yes why does not work with helmet if you need a probe to craft the helmet :S

**Comment by jusitnboggs** [comment]
I request this too for 1.12+
